stable_baselines3/common/preprocessing.py

```diff
@@ -6,6 +6,19 @@
 from torch.nn import functional as F
 
 
+def is_image_space_channels_first(observation_space: spaces.Space) -> bool:
+    """
+    Check if an image observation space (see ``is_image_space``)
+    is channels-first (CxHxW, True) or channels-last (CxHxW, False).
+
+    Use a heuristic that a channel-dimension is the smallest of the three.
```

Comment:
you are actually not checking that...

Reply:
Oh wow proper brainderp here. Will fix this and add tests.